### PR TITLE
Update going further

### PR DESCRIPTION
**Pull request by sozercan**
* Updated links
* Remove next step
* added link to k8s autoscaler

@wbuchwalter @ritazh should we remove link to acs-engine autoscaler and article?

### CODE REVIEW

File: 10-going-further/README.md [hunk 1]
```diff
@@ -32,21 +30,17 @@ Here are some tools and frameworks that can make it easy to deploy such a distri
...
+As we saw in module [7 - Distributed TensorFlow](../7-distributed-tensorflow/) and [8 - Hyperparameters Sweep](../8-hyperparam-sweep), being able to autoscale our Kubernetes cluster can be extremely useful.
+Indeed, automatic scale-out (adding more VMs to the cluster) would allow anyone to run any experiment they want, whenever they want without needing to involve an ops person to setup and prepare virtual machines.
+And because the cluster can also automatically scale-in by deleting idle VMs once training jobs are completed, we can keep the cost as low as possible by just paying for what we actually use.
 
 As of this writing, autoscaling is only supported on Kubernetes cluster created with [acs-engine](https://github.com/Azure/acs-engine).
 
 See the following resources to get started:
+
+* [Kubernetes Azure Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler/cloudprovider/azure)
 * [Kubernetes `acs-engine` Autoscaler](https://github.com/wbuchwalter/Kubernetes-acs-engine-autoscaler)
```
Comment (julienstroheker) [inline 10-going-further/README.md]: Please remove this one

File: 10-going-further/README.md [hunk 2]
```diff
@@ -32,21 +30,17 @@ Here are some tools and frameworks that can make it easy to deploy such a distri
...
 * [Autoscaling a Kubernetes cluster created with acs-engine on Azure](https://medium.com/@wbuchwalter/autoscaling-a-kubernetes-cluster-created-with-acs-engine-on-azure-5e24ddc6402e)
```
Comment (julienstroheker) [inline 10-going-further/README.md]: Same

File: 10-going-further/README.md (hunk 1 above)
Comment (julienstroheker) [reply 10-going-further/README.md]: > should we remove link to acs-engine autoscaler and article?

We talked with @wbuchwalter , the answer is yes.

Comment (sozercan) [reply 10-going-further/README.md]: removed

File: 10-going-further/README.md (hunk 2 above)
Comment (sozercan) [reply 10-going-further/README.md]: removed

**Pull request merged**